README.rst

```diff
@@ -469,6 +465,10 @@ Choose from the list of available rules:
 * **phpdoc_no_access** [@Symfony]
    | @access annotations should be omitted from phpdocs.
 
+* **phpdoc_no_alias_tag** [@Symfony]
```

Comment:
not really a good name tbh, given you rename things to their aliases if you want